ui/src/literature/components/DOILinkAction.jsx

```diff
@@ -1,30 +1,73 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import { Button, Menu } from 'antd';
 
 import ListItemAction from '../../common/components/ListItemAction';
 import EventTracker from '../../common/components/EventTracker';
+import DropdownMenu from '../../common/components/DropdownMenu';
+import IconText from '../../common/components/IconText';
+import ExternalLink from '../../common/components/ExternalLink';
 
 class DOILinkAction extends Component {
+  static renderLinkForDoi(doi) {
+    return `//doi.org/${doi.get('value')}`;
+  }
+
+  static renderDoi(doi) {
```

Comment:
we need a `DOI` component that will be used also in `DOIList`. You can see the code duplication if you check that one.

This way actually you don't even need `EventTracker` specifically here.